<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Second</class>
 <widget class="QDialog" name="Second">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1201</width>
    <height>751</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QToolButton" name="tbnLetsstart">
   <property name="geometry">
    <rect>
     <x>1000</x>
     <y>650</y>
     <width>190</width>
     <height>90</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/new/prefix1/resource/xialou.png</normaloff>:/new/prefix1/resource/xialou.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>190</width>
     <height>90</height>
    </size>
   </property>
   <property name="autoRaise">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QSlider" name="SdBGM">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>30</y>
     <width>200</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="value">
    <number>100</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="SdGame">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>90</y>
     <width>200</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="value">
    <number>100</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QToolButton" name="tbtnback">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>650</y>
     <width>190</width>
     <height>90</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/new/prefix1/resource/fanhui.png</normaloff>:/new/prefix1/resource/fanhui.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>190</width>
     <height>90</height>
    </size>
   </property>
   <property name="autoRaise">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="Leduser">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>680</y>
     <width>241</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 18pt &quot;华文中宋&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="maxLength">
    <number>20</number>
   </property>
  </widget>
  <widget class="QToolButton" name="tbtnUserclear">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>680</y>
     <width>70</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/new/prefix1/resource/userclear.png</normaloff>:/new/prefix1/resource/userclear.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>70</width>
     <height>40</height>
    </size>
   </property>
   <property name="autoRaise">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QToolButton" name="tbtnUseryes">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>680</y>
     <width>70</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/new/prefix1/resource/useryes.png</normaloff>:/new/prefix1/resource/useryes.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>70</width>
     <height>40</height>
    </size>
   </property>
   <property name="autoRaise">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btnBGM">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>30</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>默认</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BtnGame">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>90</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>默认</string>
   </property>
  </widget>
  <widget class="QToolButton" name="tbtnp1yes">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>360</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/new/prefix1/resource/yesplayer1.png</normaloff>:/new/prefix1/resource/yesplayer1.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
   <property name="autoRaise">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QToolButton" name="tbtnp2yes">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>360</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/new/prefix1/resource/yesplayer2.png</normaloff>:/new/prefix1/resource/yesplayer2.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
   <property name="autoRaise">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QToolButton" name="tbtnp3yes">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>360</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/new/prefix1/resource/yesplayer3.png</normaloff>:/new/prefix1/resource/yesplayer3.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
   <property name="autoRaise">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QToolButton" name="tbtnp4yes">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>360</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/new/prefix1/resource/yesplayer4.png</normaloff>:/new/prefix1/resource/yesplayer4.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
   <property name="autoRaise">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QToolButton" name="tbtnp5yes">
   <property name="geometry">
    <rect>
     <x>1080</x>
     <y>360</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/new/prefix1/resource/yesplayer5.png</normaloff>:/new/prefix1/resource/yesplayer5.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
   <property name="autoRaise">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QSlider" name="Sdgameclass">
   <property name="geometry">
    <rect>
     <x>910</x>
     <y>30</y>
     <width>160</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>250</number>
   </property>
   <property name="maximum">
    <number>1000</number>
   </property>
   <property name="value">
    <number>500</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="btngameclass">
   <property name="geometry">
    <rect>
     <x>1110</x>
     <y>30</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>默认</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
